server/app.py

```diff
@@ -868,8 +868,10 @@ def customer_subscription_created(event):
     # It starts by looking for a matching Contact (or creating one).
     subscription = event["data"]["object"]
     donation_type = subscription["metadata"]["donation_type"]
-    latest_invoice = subscription["latest_invoice"]
-    invoice = None
+    invoice = stripe.Invoice.retrieve(subscription["latest_invoice"])
```

Comment:
If stripe.Invoice.retrieve() doesn't find anything, it will end up sending us a message like the ones we already get, now and again, that look like "Issue finding invoice for pi_xxxxx with message...", so we'll be alerted enough to know to do something about it.